### PR TITLE
Submit listing

### PR DESCRIPTION
**Pull request by MelvinMallari**
Frontend for submit listing button

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "vim.easymotionMarkerFontFamily": "consolas"
+}
```
Comment (robrobbins) [inline .vscode/settings.json]: you need to exclude this from the repo. use the .gitignore

File: src/components/datatrust/StartListingButton.vue [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      startListing() {
+        const flashesModule = getModule(FlashesModule, this.$store)
+        const flash = new Flash('Starting Listing', FlashType.info)
+        flashesModule.append(flash)
+      },
+    },
+  })
+</script>
+
+<style scoped lang='sass'>
```
Comment (robrobbins) [inline src/components/datatrust/StartListingButton.vue]: move to an actual .sass file in the right place

File: src/components/datatrust/StartListingButton.vue [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  import FlashesModule from '../../modules/FlashesModule'
+  import { getModule } from 'vuex-module-decorators'
+  import Flash from '../../models/Flash'
+  import { FlashType } from '../../models/Flash'
+
+  export default Vue.extend({
+    methods: {
+      startListing() {
+        const flashesModule = getModule(FlashesModule, this.$store)
+        const flash = new Flash('Starting Listing', FlashType.info)
```
Comment (robrobbins) [inline src/components/datatrust/StartListingButton.vue]: let's not use hard coded strings. pretty sure i started a `constants` file

Comment (thirtyaughtsix) [reply src/components/datatrust/StartListingButton.vue]: Probably worth a ticket to track the rest of these down and 'retire' them like unlicensed, off-world replicants.  Godspeed Rutger Hauer.

Comment (robrobbins) [reply src/components/datatrust/StartListingButton.vue]: <sound of a pistol racking>

**Pull request merged**